client/components/telecom/telephony/group/telephony-group.factory.js

```diff
@@ -157,6 +157,30 @@ angular.module("managerApp").factory("TelephonyGroup", function ($q, OvhApiTelep
         });
     };
 
+
+    TelephonyGroup.prototype.fetchService = function (serviceName) {
+        var self = this;
+        var number;
+
+        // TODO : handle when group is form other service than alias
+        return OvhApiTelephony.Number().Lexi().get({
+            billingAccount: self.billingAccount,
+            serviceName: serviceName
+        }).$promise.then(function (numberOptions) {
+            number = new TelephonyGroupNumber(angular.extend(numberOptions, {
+                billingAccount: self.billingAccount
+            }));
+
+            if (self.numbers.indexOf(number)) {
```

Comment:
You don't manage if `indexOf` return is `-1`. 
Use `if (self.getNumber(number.serviceName))` as it's already implemented.